persistence/album_repository.go

```diff
@@ -263,15 +263,18 @@ func (r *albumRepository) refresh(ids ...string) error {
 	return err
 }
 
-// Return the first non empty comment, if any
 func getComment(comments string, separator string) string {
 	cs := strings.Split(comments, separator)
+	if len(cs) == 0 {
+		return ""
+	}
+	first := cs[0]
 	for _, c := range cs {
-		if c != "" {
-			return c
+		if first != c {
```

Comment:
Just a small nitpick: We don't need to compare `cs[0]` with `first` as it will all be the same. We should range the slice from the second element on.

Reply:
Fixed.